src/main/java/org/onebusaway/alexa/AnonSpeechlet.java

```diff
@@ -157,6 +159,38 @@ public SpeechletResponse onIntent(IntentRequest request, Session session)
             return setStopNumber(
                     stopNumberStr,
                     session);
+        } else if (YES.equals(intent.getName())) {
+            ArrayList<ObaStop> stops = (ArrayList<ObaStop>) session.getAttribute(FOUND_STOPS);
```

Comment:
Yeah, I agree.  Have you seen any solutions out there yet that try to target this?